### PR TITLE
Add updateStrategy for controller and speaker

### PR DESCRIPTION
**Pull request by farodin91**

### CODE REVIEW

File: charts/metallb/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # metallb
 
-![Version: 0.9.6](https://img.shields.io/badge/Version-0.9.6-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: v0.9.6](https://img.shields.io/badge/AppVersion-v0.9.6-informational?style=flat-square)
+![Version: 0.12.1](https://img.shields.io/badge/Version-0.12.1-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: v0.12.1](https://img.shields.io/badge/AppVersion-v0.12.1-informational?style=flat-square)
```
Comment (gclawes) [inline charts/metallb/README.md:3]: Same as in #1369, it looks like there are missing fields in the `README.md` that are getting pulled in here.  @fedepaol  we may need to look at the release process and possibly automate the chart `README.md` generation

Comment (fedepaol) [reply charts/metallb/README.md:3]: Yeah automating and enforcing would be great.
We are doing it already for a bunch of other resources, CI won't allow the PR in if the content is not aligned.

Comment (fedepaol) [reply charts/metallb/README.md:3]: Filed https://github.com/metallb/metallb/issues/1373 to track this

**Comment by fedepaol** [comment]
@farodin91 mind rebasing? We now have automatic readme generation / enforcing.
Thanks!

**Comment by farodin91** [comment]
@fedepaol I have rebased some time ago.

**Comment by fedepaol** [comment]
> @fedepaol I have rebased some time ago.

I still see a conflict though, can you fix it? Thanks!

**Comment by farodin91** [comment]
@fedepaol Updated.

**Comment by fedepaol** [comment]
LGTM, thanks

**Pull request merged**